File: chaincode/example-java/src/main/java/org/hyperledger/fabric/example/Asset.java
package org.hyperledger.fabric.example;

import com.google.gson.JsonObject;
import org.hyperledger.fabric.shim.Chaincode;
import org.hyperledger.fabric.shim.ChaincodeStub;

import java.util.HashMap;

public interface Asset {
    public String className();
    public HashMap<String,String> ownerShip();
    public Chaincode.Response invoke(Parameter params) throws Exception;
}

File: chaincode/produce_tracing2/asset.go
package main

import "github.com/hyperledger/fabric/protos/peer"

type Asset interface {
	// 资产的类名
	Class() string
	// 资产所属
	Ownership() map[string]string
	// 方法调用
	Invoke(params Parameter) peer.Response
}
